src/main/java/org/casbin/jcasbin/model/Model.java

```diff
@@ -14,19 +14,19 @@
 

```

Comment:
FYI, this file uses `CRLF`.

Reply:
Hi @dimi-nk, I noticed some files are using CRLF, can you make a PR to fix them all into LF?

Reply:
Done in https://github.com/casbin/jcasbin/pull/79

Reply:
Thanks! Merged.